sql/core/src/test/resources/sql-tests/inputs/literals.sql

```diff
@@ -107,7 +107,11 @@ select integer '2147483648';
 
 -- awareness of the negative/positive sign before type
 select -integer '7';
+select +integer '7';
+select +date '1999-01-01';
+select +timestamp '1999-01-01';
+select +x'2379ACFe';
+-- can't negate date/timestamp/binary
 select -date '1999-01-01';
 select -timestamp '1999-01-01';
 select -x'2379ACFe';
-select +integer '7';
```

Comment:
Does this case not supported?

Reply:
it's supported: https://github.com/apache/spark/pull/26578/files#diff-3adf2824cec2162e632a5114d3d710cdR110